modules/csg/csg_shape.h

```diff
@@ -89,12 +89,14 @@ class CSGShape3D : public GeometryInstance3D {
 		Vector<Vector3> normals;
 		Vector<Vector2> uvs;
 		Vector<real_t> tans;
+		Vector<int> indices;
 		Ref<Material> material;
 		int last_added = 0;
 
 		Vector3 *verticesw = nullptr;
 		Vector3 *normalsw = nullptr;
 		Vector2 *uvsw = nullptr;
+		int *indexsw = nullptr;
```

Comment:
```suggestion
		int *indicesw = nullptr;
```
To match the vector, the other pointers match their vectors, I'd also move it down one to match the order